### PR TITLE
Update the copyright year to 2019

### PR DESCRIPTION
**Pull request by kestewart**
Update all the copyright years to 2019.

### CODE REVIEW

File: chapters/index.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
...
\ No newline at end of file
+With thanks to Adam Cohn, Andrew Back, Ann Thornton, Bill Schineller, Bruno Cornec, Ciaran Farrell, Daniel German, David Wheeler, Debra McGlade, Dennis Clark, Ed Warnicke, Eran Strod, Eric Thomas, Esteban Rockett, Gary O'Neall, Guillaume Rousseau, Hassib Khanafer, Jack Manbeck, Jaime Garcia, Jeff Luszcz, Jilayne Lovejoy, John Ellis, Karen Copenhaver, Kate Stewart, Kevin Mitchell, Kim Weins, Kirsten Newcomer, Kris Reeves, Liang Cao, Marc-Etienne Vargenau, Mark Gisi, Marshall Clow, Martin Michlmayr, Martin von Willebrand, Matt Germonprez, Michael J. Herzog, Michel Ruffin, Nuno Brito, Oliver Fendt, Paul Madick, Peter Williams, Phil Robb, Philip Odence, Philip Koltun, Phillippe Ombredanne, Pierre Lapointe, Rana Rahal, Robin Gandhi, Sam Ellis, Sameer Ahmed, Scott K Peterson, Scott Lamons, Scott Sterling, Shane Coughlan, Steve Cropper, Stuart Hughes, Tom Callaway, Tom Vidal, Thomas F. Incorvia, Thomas Steenbergen, Venkata Krishna, W. Trevor King, Yev Bronshteyn, and Zachary McFarland for their contributions and assistance.
```
Comment (tsteenbe) [inline chapters/index.md]: This block should not have changed , think accidental white space

File: chapters/1-rationale.md [hunk 2]
```diff
@@ -88,4 +88,4 @@ Companies and organizations (collectively “Organizations”) are widely using
 
 **1.8.5** A new appendix “Using SPDX short identifiers in Source Files” has been added to document the best practices to refer to the licenses in the SPDX license list that have emerged from the development community. See [Appendix V: Using SPDX short identifiers in Source Files](./appendix-V-using-SPDX-short-identifiers-in-source-files.md) for more information.
 
-**1.8.6** Miscellaneous bug fixes as reported on the mailing list and reported as issues on the [spdx-spec GitHub repository](https://github.com/spdx/spdx-spec).
\ No newline at end of file
+**1.8.6** Miscellaneous bug fixes as reported on the mailing list and reported as issues on the [spdx-spec GitHub repository](https://github.com/spdx/spdx-spec).
```
Comment (tsteenbe) [inline chapters/1-rationale.md]: This should not have changed when changing only the year

File: chapters/index.md (hunk 1 above)
Comment (kestewart) [reply chapters/index.md]: This is caused by adding POSIX style line endings.

**Comment by kestewart** [comment]
Change milestone for this to 2.2.     Need to unblock 2.1.1.

File: chapters/1-rationale.md (hunk 2 above)
Comment (kestewart) [reply chapters/1-rationale.md]: Agree.   It should not have been changed.

File: chapters/index.md (hunk 1 above)
Comment (kestewart) [reply chapters/index.md]: There are now new authors added for 2.2.

File: chapters/1-rationale.md (hunk 2 above)
Comment (kestewart) [reply chapters/1-rationale.md]: Went in and did some manual edits on the pull.   Should be fine now.  If not, please flag as an issue.

**Comment by kestewart** [comment]
@tsteenbe - can you remove your requested change flags.   I think we've sorted the outstanding issues, and are trying to move this forward to be 2.2-DRAFT so we can be pulling in changes folks are making.  2.1.1 was published a couple of months ago.

**Comment by tsteenbe** [comment]
Merged in https://github.com/spdx/spdx-spec/commit/133eac5dbd12f8aced5224756c31fe63caaceab1

**Pull request closed without merging**